src/utils/tracked-controls.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Return enumerated gamepads matching id prefix.
+ *
+ * @param {object} idPrefix - prefix to match in gamepad id, if any.
+ */
+module.exports.getGamepadsByPrefix = function (idPrefix) {
+  var gamepadsList = [];
+  var gamepad;
+  var gamepads = navigator.getGamepads && navigator.getGamepads();
+  if (!gamepads) { return gamepadsList; }
+
+  for (var i = 0; i < gamepads.length; ++i) {
+    gamepad = gamepads[i];
+    if (gamepad) {
+      if (!idPrefix || gamepad.id.indexOf(idPrefix) === 0) {
+        gamepadsList.push(gamepad);
+      }
+    }
+  }
+  return gamepadsList;
+};
+
+/**
+ * Enumerate controllers (as built by system tick, e.g. that have pose) and check if they match parameters.
+ *
+ * @param {object} idPrefix - prefix to match in gamepad id, if any.
+ * @param {object} queryObject - map of values to match (hand; index among controllers with idPrefix)
+ */
+module.exports.isControllerPresent = function (idPrefix, queryObject) {
+  var isPresent = false;
+  var index = 0;
+  var gamepad;
+  var isPrefixMatch;
+  var sceneEl = document.querySelector('a-scene');
+  var gamepads = null;
```

Comment:
Don't need to initialize a null. The `undefined` default value is already falsy.